src/_codelabs/index.md

```diff
@@ -6,23 +6,82 @@ permalink: /codelabs
 toc: false
 ---
 
-See the latest Dart codelabs on the Google Developers site:
+The latest Dart codelabs are on the Google Developers site.
 
-**General**
-* [Intro to Dart for Java Developers](https://codelabs.developers.google.com/codelabs/from-java-to-dart/)
+## General
```

Comment:
This page looks much better, but it feels a little cramped because of the h2 headings.

Consider a `margin-bottom: 1em` to `h2`, possibly by using this custom styling at the top of the page:
```html
<style> main article .content h2 { margin-bottom: 1em; }</style>
```